x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/planner/QueryTranslatorFailTests.java

```diff
@@ -205,35 +205,4 @@ public void testStringContainsWrongParams() {
         assertEquals("1:15: second argument of [stringContains(process_name, 1)] must be [string], found value [1] type [integer]",
                 error("process where stringContains(process_name, 1)"));
     }
-
-    public void testWildcardNotEnoughArguments() {
```

Comment:
Please convert these to `like` if there aren't tests that already do that.

Reply:
These tests become now syntactic failures. One case could however be indeed converted (ex `testWildcardWithNumericField`).

Reply:
As long as we have equivalent semantical tests, I'm 👍 . Note that wildcard tests that rely on the syntax can be changed to `like/like~`  hence my point of replacing removed tests (or checking) there are semantical equivalents to them.